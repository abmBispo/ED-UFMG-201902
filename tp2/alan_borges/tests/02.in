1000 3 1
500 a
100 b
300 c