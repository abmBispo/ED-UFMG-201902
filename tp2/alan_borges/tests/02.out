1 a 500
1 b 100
1 c 300
